README.md

```diff
@@ -17,22 +17,20 @@ The Kubeflow pipelines service has the following goals:
 
 ## Installation
 
-* Install Kubeflow Pipelines from choices described in [Installation Options for Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/installation/overview/).
+* Install Kubeflow Pipelines from choices described in [V1-Installation Options for Kubeflow Pipelines](https://www.kubeflow.org/docs/components/pipelines/legacy-v1/installation/overview/) and [V2-Operator Guides Installation](https://www.kubeflow.org/docs/components/pipelines/operator-guides/installation/).
```

Comment:
I wonder if we shouldn't use https://www.kubeflow.org/docs/components/pipelines/operator-guides/installation/ instead. I understand though it lacks a lot of information and the info in the v1 docs helps a lot.
I'd like to know the opinion of others.

cc @HumairAK @rimolive @gregsheremeta

Reply:
I think this section should call out that you can install Kubeflow as a whole, or you can install Kubeflow (v2) as a standalone. So yeah, I think @hbelmiro 's suggested [link](https://www.kubeflow.org/docs/components/pipelines/operator-guides/installation/) makes the most sense (although I'd copy a bit of the blurb from that page too).

I'm generally not a fan of still linking to old v1 stuff from v2 docs. v1 should be relegated to its hidden corner of the docs.

Reply:
Thanks @hbelmiro @gregsheremeta!
I have updated this is a new commit which I can squash when the PR is ready.